### DISCUSSION TITLE
2024/08/25/10/

### DISCUSSION THREAD

**Discussion by giscus** [MengZe2l/mengze2l.github.io#6] [General]
# 2024/08/25/10/

我不是换hexo了吗，一直是使用Spck Editor写文章，每次写好文章都要push一次，非常麻烦，最近在群友那了解到有Github Action自动化部署，可以写好文章自动部署到github pages，而且github的编辑器也是非常好用的，写好提交就能自动发送，–怎么有点像SSR了–，于是我就开始了这一个半小时的折腾…没有电脑，使用安卓设备折腾！ 如果你没有Hexo博客，那么请看这里，如果

https://www.mengze2.cn/2024/08/25/10/

**Comment by MengZe2l** [comment]
非常棒了，我本站已经安排上
